test/spec/renderer_spec.js

```diff
@@ -143,6 +146,9 @@ describe('Renderer', function () {
         id: 1,
         adUnitCode: undefined
       });
+      expect(loadExternalScript.called).to.be.false;
+
+      testRenderer.render()
```

Comment:
The script is effectively not loaded called before a call to `render` but loading start afterwards